src/locale/languages/tr.js

```diff
@@ -0,0 +1,318 @@
+export const tr = {
+  auth: {
+    login: {
+      welcomeTitle: 'GitPoint\'e Hoşgeldiniz',
+      welcomeMessage: 'En zengin özelliklere sahip GitHub client\'ı ve 100% ücretsiz',
+      notificationsTitle: 'Bildirimleri Kontrol Et',
+      notificationsMessage:
+        'Okunmamış ve katılımcı olduğunuz bildirimlerinizin tümünü görüntüleyin ve kontrol edin',
+      reposTitle: 'Repository\'ler ve Kullanıcılar',
+      reposMessage:
+        'Repository\'leri, kullanıcıları ve organizasyon bilgilerini kolayca edinin',
+      issuesTitle: 'Issue\'lar ve Pull Request\'ler',
+      issuesMessage:
+        'Sohbet ederek iletişim kurun, pull request\'leri merge edin ve daha fazlası',
+      signInButton: 'GİRİŞ YAP',
+    },
+    welcome: {
+      welcomeTitle: 'GitPoint\'e Hoşgeldiniz',
+    },
+    events: {
+      welcomeMessage:
+        "Hoşgeldiniz! Bu, haber kaynağınızdır - izlediğiniz repository'lerde ve takip ettiğiniz kişilerle ilgili son etkinlikleri takip etmenize yardımcı olur.",
+      commitCommentEvent: 'commit\'e yorum yapıldı',
+      createEvent: '{{object}} oluşturuldu',
+      deleteEvent: '{{object}} silindi',
+      issueCommentEvent: '{{action}} on {{type}}',
+      issueEditedEvent: '{{action}} their comment on {{type}}',
+      issueRemovedEvent: '{{action}} their comment on {{type}}',
+      issuesEvent: '{{action}} issue',
+      publicEvent: {
+        action: 'made',
+        connector: 'public',
+      },
+      pullRequestEvent: '{{action}} pull request',
+      pullRequestReviewEvent: '{{action}} pull request review',
+      pullRequestReviewCommentEvent: '{{action}} on pull request',
+      pullRequestReviewEditedEvent: '{{action}} their comment on pull request',
```

Comment:
> their comment on 

This is a lot where there is and not translated, will remain in English?

Reply:
I haven't any idea about the Turkish meaning of these descriptions. I need a little time in order to translate these descriptions. It can be used for a while in that type. So, it will remain in English for a while.

Reply:
I got you, usually and not translated "specific" Git / GitHub terms (not always, but so often
), such as `pull request`, `merge`, and even `issue`. But expressions like "their comment on" better translate.

Reply:
I will work on PR as soon as possible.